### PR TITLE
SONiC DHCP Snooping model definition

### PR DESCRIPTION
**Pull request by simon-nathans**
The DHCP Snooping model definition for SONiC.

### CODE REVIEW

File: models/enterprise_sonic/dhcp_snooping/dhcp_snooping.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    - "Tested against Enterprise SONiC Distribution by Dell Technologies."
+  short_description: "Manage DHCP Snooping on SONiC"
+  description: "This module provides configuration management of DHCP snooping for devices running SONiC."
+  author: Simon Nathans (@simon-nathans)
+  options:
+    config:
+      description: The DHCP snooping configuration.
+      type: dict
+      suboptions:
+        afis:
```
Comment (kerry-meyer) [inline models/enterprise_sonic/dhcp_snooping/dhcp_snooping.yaml:26]: The "afis" dict should also contain an "enabled" bool for each AFI.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
The structure and content of this model look good.

I have posted a few comments requesting minor changes and also am requesting the addition of a global "per-afi" attribute to indicate the "enabled" state for DHCP snooping on that AFI. (Please let me know if there is some reason why this is not needed.)

**Review by kerry-meyer — APPROVED** [review]
Thanks for making the additional requested changes.

Approved.

**Comment by simon-nathans** [comment]
Closing because commits were not made on a new branch. Will open a new PR

**Pull request closed without merging**